cmd/admin-server-info.go

```diff
@@ -242,7 +243,7 @@ func getLocalServerProperty(endpointZones EndpointZones, r *http.Request) madmin
 	return madmin.ServerProperties{
 		State:    "ok",
 		Endpoint: addr,
-		Uptime:   UTCNow().Sub(globalBootTime),
+		Uptime:   uint64(UTCNow().Sub(globalBootTime) / time.Second),
```

Comment:
If you are sending uint64 then you should send UTCNow().Unix()-globalBootTime.Unix()

Reply:
done